lib/cosmos/tools/cmd_tlm_server/cmd_tlm_server_config.rb

```diff
@@ -153,7 +153,7 @@ def process_file(filename, recursive = false)
             current_interface_or_router.name = interface_name
             @interfaces[interface_name] = current_interface_or_router
 
-          when 'LOG', 'DONT_LOG', 'TARGET'
+          when 'LOG', 'LOG_STORED', 'DONT_LOG', 'TARGET'
```

Comment:
LOG_STORED is to specify a different log writer for stored telemetry? What happens if you don't have this? What is the default for stored telemetry?

Reply:
My default stored telemetry is intermingled in the normal log files.  If you add LOG_STORED then it will go into a different file.